libs/usb-drive/src/usb_drive.ts

```diff
@@ -0,0 +1,143 @@
+import { promises as fs } from 'fs';
+import { join } from 'path';
+import { execFile } from 'child_process';
+import { promisify } from 'util';
+import { assertDefined } from '@votingworks/basics';
+import makeDebug from 'debug';
+
+const exec = promisify(execFile);
+
+const debug = makeDebug('usb-drive');
+
+export type UsbDriveStatus =
+  | { status: 'no_drive' }
+  | { status: 'mounted'; mountPoint: string }
+  | { status: 'ejected' }
+  | { status: 'error'; reason: 'bad_format' };
```

Comment:
I don't think so and here's my reasoning:
- I don't think we have a need to tell users when mounting is in progress. Pretty sure we don't really do this today. Though if you have a diff impression I'm totally open to hearing it. I could be convinced it's worth telling users about if it's particularly slow - like "hey we saw you inserted a USB drive and we're checking it out"
- We can show an "ejecting in progress" state to users by waiting for the promise to complete

Reply:
re: mounting, we do show this in a couple places. If you are looking at the USB button in the top right corner of VxAdmin or VxCentralScan, it will show a mounting state when mounting. I don't think this is terribly important. But we also show it in modals that require USBs. If you open the modal without a USB drive, it will show the insert USB drive icon and when you do insert, will give feedback that it's mounting. I've always felt that the earlier feedback is nice-to-have and avoids the user having a momentary "it's not working" moment. but I can't remember how long the mounting takes exactly - maybe it wouldn't be noticeable to the user

re: ejecting, that seems fine. I wonder if it will be annoying to pass or convert promises into props for UI components that need ejecting feedback, but if you find it's not a problem then I think better to have one less status.

Reply:
Ok that's helpful context about mounting. I'm game to remove it and see if it feels laggy, or just support it. I could go either way. Mounting seems pretty much instantaneous in my VM, which is why I didn't originally think to support it.

Re: ejecting, I wouldn't use the promise directly, I would use the react-query mutation `isLoading` flag.

Reply:
I think this is a good question for @mattroe who may have locked down machines on hardware to see how long it takes in production conditions.

@mattroe do you have locked down VxAdmin, or VxDev VxAdmin? If you go to a modal that requires a USB and then insert it, how long does it show the "Loading..." message for? Could we go without it?

Reply:
@adghayes I don't see any Loading screen when testing on a locked down VxAdmin. The screen just changes state within what seems like less than a second.

Reply:
Huh, that's not what I remember, but it seems like enough of an indication that it's OK to remove.

Reply:
Thanks! We can keep an eye on it